src/NuProj.Tasks/Extensions.cs

```diff
@@ -143,5 +157,24 @@ public static void UpdateMember<T>(this T target, Expression<Func<T, string>> me
             
             property.SetValue(target, list, null);
         }
+
+        public static string Combine(this PackageDirectory packageDirectory, string targetFramework, string fileName)
+        { 
+            switch (packageDirectory)
+            {
+                case PackageDirectory.Root:
+                    return fileName;
+                case PackageDirectory.Content:
+                    return Path.Combine(Constants.ContentDirectory, fileName);
+                case PackageDirectory.Build:
+                    return Path.Combine(Constants.BuildDirectory, targetFramework, fileName);
```

Comment:
This might be the line responsible for my ProjectReference with PackageDirectory=Build going into the build\net451 directory instead of just "build". Is this an oversight, or did you intend to to put the project reference output into a targetFramework subfolder?
In my scenario, I have a .targets file in the build folder and I need to pick up the DLL next to it. But if it's in a subfolder, my .targets file now has to be aware of that. It certainly *can* be, but I wonder why it's necessary.